apps/themovie/src/pages/Login/index.js

```diff
@@ -11,6 +11,36 @@ import {
 } from '../styled-components-example/styles';
 
 export default class Section extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			email: '',
+			password: ''
+		};
+	}
+
+	emailChange = event => {
+		this.setState({ email: event.target.value });
+	};
+
+	passChange = event => {
+		this.setState({ password: event.target.value });
+	};
```

Comment:
Como a emailChange e passChange são bem parecidas, não seria bom transformar elas em apenas 1 function?
```suggestion
  handleChange = event => {
    const { value, name } = event.target;

    this.setState({
      [name]: value
    });
  }
```

Reply:
Ajeitei la

Reply:
Arrumei lá :D 👍